README.md

```diff
@@ -14,20 +14,20 @@ name.
 This is a two stage process. The code first uses Reshuffle's [Entertainment
 Identifier Registry (EIDR)](https://eidr.org/) connector to resolve the
 movie name into a standaridized movie ID. It then uses the
-[Movies Anywhere](https://moviesanywhere.com/) to lookup to movie poster
+[Movies Anywhere](https://moviesanywhere.com/) public API to lookup a movie poster
 image given this ID.
 
 #### Usage
 
-First install the example and start the Reshuffle server:
+Install this example and start it:
 
 ```bash
 npm install reshuffle-movieposter-example
 npm run start
```

Comment:
this is wrong!
examples need to go to reshuffle package to avoid a circular dependency.
Need to remove reshuffle from this package dependency.

Reply:
@avnerbraverman ?